Yakkity-Yulelogs/reactive-nutshell
# TASKS - Mark As Complete Bug

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

I fixed an issue where editing a task was resetting the task's completion status to false.

Please delete options that are not relevant.


# Testing Instructions

```
git fetch --all
git checkout gc-taks-editformbug 
```

Part 1:

- Switch to Tasks view
- Check any of the "Mark as completed" checkboxes 
- Click on Edit button of that same task
- Edit something and save the change
- Expect to see the task being updated with the new info
- Expect the completion checkbox to stay the same

Part 2:

- Switch to Tasks view
- Uncheck any of the "Mark as completed" checkboxes 
- Click on Edit button of that same task
- Edit something and save the change
- Expect to see the task being updated with the new info
- Expect the completion checkbox to stay the same
- 

# Checklist:
